lib/Metadata.php

```diff
@@ -82,6 +82,25 @@ public function getFieldMappings()
     }
 
     /**
+     * Return the mapping for the named field.
+     *
+     * @see Metadata::addFieldMapping
```

Comment:
And what is this `@see` about? You are referencing to the method itself here, aren't you?